src/artofillusion/polymesh/MeshUnfolder.java

```diff
@@ -1286,1163 +788,52 @@ private UnfoldedMesh computeUnfoldedMesh(ArrayList<Integer> vertList,
 		return new UnfoldedMesh(vertices, edges, faces);
 	}
 
-	private UnfoldedMesh computeUnfoldedMesh1(ArrayList<Integer> vertList,
-			ArrayList<Integer> edgeList, ArrayList<Integer> faceList,
-			UnfoldedVertex[] uverts, UnfoldedEdge[] uedges,
-			UnfoldedFace[] ufaces) throws IterativeSolverNotConvergedException {
-		UnfoldedVertex[] vertices = new UnfoldedVertex[vertList.size()];
-		UnfoldedEdge[] edges = new UnfoldedEdge[edgeList.size()];
-		UnfoldedFace[] faces = new UnfoldedFace[faceList.size()];
-		int[] vertTrans = new int[uverts.length];
-		int[] edgeTrans = new int[uedges.length];
-		int[] faceTrans = new int[ufaces.length];
-		// first, translate the indexes
```

Comment:
Was this support code for the other method?

Reply:
computeUnfoldedMesh() still there )))